### ISSUE TITLE
Enrichment cannot create IRIs

### ISSUE THREAD

**Issue by wouterbeek** [pldn/LDWizard#145]
When we use a SPARQL query to enrich the ID in the following source table to an IRI:

| BAG ID |
| --- |
| 123 |

We get the following linked data:

```turtle
_:b0 <{baseIri}/BAG%20ID> "123" .
_:b0 <{baseIri}/BAG%20ID> "https://data.kkg.kadaster.nl/id/gebouw/123" .
```

But we expect:

```turtle
_:b0 <{baseIri}/BAG%ID> <https://data.kkg.kadaster.nl/id/parcel/456> .
```

**Comment by philipperenzen** [comment]
Currently, the built in `Convert to IRI` column refinement transformation will do this, it will replace the original value in the RDF file - not in the CSV file. A custom column refinement transformation will not do this, instead it will create a `columnName-refined` triple and and header in the CSV file.

Here is an example GIF of the current version of the repository, with a custom Column refinement that will add "-abc" at the end of the input value. I am using `yarn dev` for local development with the custom function set up in the "runtimeConfig.ts" file.

![Screen Recording 2023-09-08 at 11 29 58](https://github.com/pldn/LDWizard/assets/112689808/91a79ce4-4d99-4720-a6af-8321491cbb7d)

[LDWExample.csv](https://github.com/pldn/LDWizard/files/12557998/LDWExample.csv)

<details>
  <summary>Click to expand custom config</summary>

    const wizardConfig = {
       appName: "LDWizard - example",
       defaultBaseIri: "https://w3id.org/my-wizard/",
       primaryColor: "#4caf50", // green
       secondaryColor: "#1565c0", // blue
       columnRefinements: [
          {
             label: "Add -abc at the end",
             type: "single" as const,
             description:
                "This transformation will take input value and add '-abc' at the end",
             transformation: async (term: string) => {
                return `${term}-abc`
             },
          },
       ],
    };
    
    export default globalThis.wizardConfig = wizardConfig

</details>

**Comment by philipperenzen** [comment]
This has been resolved in version 3.0.0: 
* by default LDWizard will return an IRI that uses the default IRI (unless specified otherwise) with the csv value at the end, like it used to. This would look somewhat like this after the configuration step: `https://data.pldn.nl/8f6052/def/{csv_value}`
* within column refinements it can be specified if a transformation should return an IRI or literal with `yieldsLiteral` or `yieldsIri`.

Please feel free to reopen this issue if it persists.
